VTUL/dlp-access
# load contact component details from config json

**Load the contents of the contact section from config json files. Don't render section at all if config is not set.**
* * *

**JIRA Ticket**: (https://webapps.es.vt.edu/jira/browse/LIBTD-2062) (:star:)

* Other Relevant Links (Meeting note, project page, related pull requests, etc.)

# What does this Pull Request do? (:star:)
Load the contents of the contact section from config json files. Don't render section at all if config is not set.

# What's the changes? (:star:)
* Add `public/site_data` to .gitignore so that I can test with config files locally
* Add a contact section to each config to define its content
* Refactor ContactSection component to read from config instead of just rendering hardcoded copy

# How should this be tested?
* start server locally like this: `REACT_APP_REP_TYPE=IAWA REACT_APP_CONFIG_PATH="https://vtdlp-dev-site-data.s3.amazonaws.com" npm start`
* Make sure that the IAWA contacts render properly
* Stop server and re-start like this: `REACT_APP_CONFIG_PATH="https://vtdlp-dev-site-data.s3.amazonaws.com" npm start`
* Check that the site falls back to the default VTDLP content and that there is contact info there for us

# Additional Notes:
* branch: `LIBTD-2062`

# Interested parties
@yinlinchen 

(:star:) Required fields
